Makefile

```diff
@@ -54,7 +54,8 @@ _output/release/bootkube.tar.gz: _output/bin/linux/bootkube _output/bin/darwin/b
 	tar czf $@ -C _output bin/linux/bootkube bin/darwin/bootkube bin/linux/checkpoint
 
 run-%: GOFLAGS = -i
-run-%: clean-vm-% _output/bin/linux/bootkube _output/bin/$(LOCAL_OS)/bootkube
+run-%: clean-vm-% _output/bin/linux/bootkube _output/bin/$(GOOS)/bootkube
+	test $(GOOS) = $$(uname | tr A-Z a-z)
```

Comment:
Why this test?

Reply:
> Why this test?

I tried to motivate that with the “The `uname`/`tr` → `go env` change affects the `run-%` prereqs…” paragraph in my [topic post][1].  Is there something that needs more support in there?

I'm not terribly excited about this line myself, but I think narrowing our reliance on `uname`/`tr` is useful even if we don't remove it entirely.  I'm all for removing it entirely if you have ideas on that front :).

[1]: https://github.com/kubernetes-incubator/bootkube/pull/959#issue-179775044